### DISCUSSION TITLE
frame synchronization between previous and next sequences

### DISCUSSION THREAD

**Discussion by balthou1703** [grizzlypeak3d/DJV#499] [Ideas]
The feature that allow us to compare sequence by doing a page up/pae down is pretty cool and usefull but it would be even better if there was a frame sync between sequences: 
ex:
I load seq_0.0001-0100.jpg and seq_1.0001-0100.jpg.
I look seq_0 at frame 24, I page down and the display switch to seq_1 at frame 24.

**Comment by darbyjohnston** [comment]
Hi, and sorry for the long delay; I just released a new version, 3.0.0, that has A/B comparison. This might provide the functionality you want:
https://darbyjohnston.github.io/DJV/index.html#compare
